### PR TITLE
Update UI for Apps, Data Sources and API Keys

### PR DESCRIPTION
**Pull request by lasryaric**
Adjusted the UI of Apps / Data Sources / API Keys for consistency.
Screenshots are attached.

Screen Shot 2023-05-16 at 14 51 55
Screen Shot 2023-05-16 at 14 53 46
Screen Shot 2023-05-16 at 14 54 39
Screen Shot 2023-05-16 at 14 54 44
Screen Shot 2023-05-16 at 14 54 49
Screen Shot 2023-05-16 at 14 55 38
Screen Shot 2023-05-16 at 14 56 14

### CODE REVIEW

**Comment by lasryaric** [comment]
@gabhubert please look at these screenshots and let me know what you think.
I will change all the mention of "data source" to "Data Source" in another PR.

**Comment by spolu** [comment]
Looks great: minor tweaks:
- replace "Your Large Language Model apps" by "Build custom Large Language Model" (will work even when looking at someone else's profile)?
- Maybe add a tiny bit of margin right to the buttons so that they align with the buttons in the list (visible on data sources, and api keys sreens)

File: front/pages/w/[wId]/a/index.tsx [hunk 1]
```diff
@@ -64,22 +64,27 @@ export default function Apps({
...
-                  </div>
-                </div>
-              )}
+            <div className="mt-16 flex flex-col justify-between lg:flex-row lg:items-center">
+              <div className="">
+                <h1 className="text-base font-medium text-gray-900">Apps</h1>
 
-              <div className="mt-8 overflow-hidden">
+                <p className="text-sm text-gray-500">
+                  Your Large Language Model apps.
```
Comment (spolu) [inline front/pages/w/[wId]/a/index.tsx:72]: "Build custom Large Language Model Apps"

**Review by spolu — DISMISSED** [review]
2 minor changes otherwise feel free to merge as is :+1:

**Comment by lasryaric** [comment]
> replace "Your Large Language Model apps" by "Build custom Large Language Model" (will work even when looking at someone else's profile)? 
Text will be the same even when looking at someone else's  profile. There is a screenshot for that case in the description.

**Comment by gabhubert** [comment]
Most of my comments are in [Slack](https://dust4ai.slack.com/archives/C050A0S2Z7F/p1684248420488669?thread_ts=1683715247.932029&cid=C050A0S2Z7F) already, won't duplicate here!

I think at the meta level, anytime we push a change we should stay involved in checking "holistically" how the thing looks and feels. I know some of the points I mentioned aren't focused on the exact PR diffs, but they'll still be shown to the user. And it's fine to consider them "fix later" but having you pre-emptively mention that makes it easier to know that you've checked for details etc to get the the level of quality we want for users.

To give you a sense of how I review, the things I focus a lot on (you may have noticed 😅) are: 

**usability:**
- does the thing clearly show what's expected? 
- do the things do the thing they say they do? 

**information hierarchy:**
- are the most important things more visible? 
- are the things that are similarly important similarly visible?

**robustness:**
- what will it look like when signed in? vs signed out? 
- what will it look and feel like for platform users vs design partners? 

**consistency:** 
- consistency in spacing
Screenshot 2023-05-16 at 16 00 25
- consistency in wording
Screenshot 2023-05-16 at 16 02 42

- consistency  in grammar
Screenshot 2023-05-16 at 16 04 57


**concision:**
- can we make the text shorter without loosing meaning?

**Pull request merged**